### PR TITLE
a whole bunch of stuff

### PR DESCRIPTION
**Pull request by fostermh**
Fix https://github.com/cioos-siooc/metadata-entry-form/issues/264 
Fix https://github.com/cioos-siooc/metadata-entry-form/issues/265 
Fix https://github.com/cioos-siooc/metadata-entry-form/issues/127
Fix https://github.com/cioos-siooc/metadata-entry-form/issues/240

This PR replaces and is a combination of the following existing PR's
https://github.com/cioos-siooc/metadata-entry-form/pull/241 https://github.com/cioos-siooc/metadata-entry-form/pull/242 https://github.com/cioos-siooc/metadata-entry-form/pull/281 https://github.com/cioos-siooc/metadata-entry-form/pull/284 https://github.com/cioos-siooc/metadata-entry-form/pull/314 https://github.com/cioos-siooc/metadata-entry-form/pull/320

### CODE REVIEW

**Comment by JessyBarrette** [comment]
related to https://github.com/HakaiInstitute/hakai-data/issues/107

File: src/components/Tabs/TaxaTab.jsx [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+                        Data processing history (provenance) for the resource.
+                    </En>
+                    <Fr>
+                        Historique du traitement des données (provenance) pour la ressource.
+                    </Fr>
+                    <RequiredMark passes={validateField(record, "distribution")} />
+                    <SupplementalText>
+                        <I18n>
+                            <En>
+                                Enter Information about the events or source data used in constructing the data specified by the scope.
```
Comment (timvdstap) [inline src/components/Tabs/TaxaTab.jsx:144]: Please enter the metadata about the taxonomic areas covered by the resource.

**Comment by Br-Johnson** [comment]
Final change requests:

Change field 'Dataset Type' to 'Resource Type' and move from Data Identification Tab to Start tab.
Move Title field on the Start tab rather than Data Identification Tab
Change 'Data Identification' tab name to 'Resource Identification'
Drop 'Theme' selector on Start tab all together. Rely on resourceType to define how tabs appear.
If model is selected, then drop platform.
On the Taxonomic Classification Tab: Include an option to add 'This resource does not include taxonomic classification"
Questions:
Lineage Scope? What's that refer to? Confusing considering you can have multiple sources per lineage, AND multiple lineages.

EML vs ISO lineage and Methods Processing steps? Where does each field go? Does Methods and Processing Steps go in only EML or both EML and ISO?

See how NOAA leverages lineage for reference: https://www.ncei.noaa.gov/access/metadata/landing-page/bin/iso?id=gov.noaa.ngdc.mgg.dem:4956

**Comment by Br-Johnson** [comment]
Talked real time with Matt, Tim, and Jessy about this. Here's a summary:

- [ ] Move Title to Start tab
- [ ] Rename ‘Dataset Type’ to ‘Resource Type’
- [ ] Move Resource Type to Start tab (use options from DataCite)
- [ ] Theme to Start Page: Oceanographic, Biological, Other. Multi-select checkboxes. 
- [ ] If select other: Hide EOVs
- [ ] Add a checkbox to say no depth
- [ ] Add a checkbox to say no taxonomic classification
- [ ] Change Lineage Description to Lineage Title 
- [ ] Change default scope of lineage to whatever primary resource is. 
- [ ] Offer select checkbox to choose scope (include Data Collection (sampling)
- [ ] Change naming Lineage Scope to Resource Type
- [ ] Rename Data Identification tab to Resource Identification
- [ ] Move DOI to start tab
- [ ] Add summary number of resources in accordions on Data/Documentation tab and drop colons

**Comment by fostermh** [comment]
ok, I think this is ready for review again

**Comment by fostermh** [comment]
re resource type: so.. which ones?
re DOI buttons: if you are not on the hakai org, then yes.

**Comment by JessyBarrette** [comment]
> re resource type: so.. which ones? 

Let's create an issue for that https://github.com/cioos-siooc/metadata-entry-form/issues/345

**Comment by JessyBarrette** [comment]
Looks like now all the tabs in `Data and Documentation` are expended by default.

**Comment by fostermh** [comment]
> Looks like now all the tabs in `Data and Documentation` are expended by default.

only if they are empty. I think they will get hidden and ignored otherwise.

**Comment by JessyBarrette** [comment]
Ok I think I only two more comments:

1. Title should be first on the start page
2. Perhaps we can default to `Dataset` within the resource type dropdown

**Comment by fostermh** [comment]
ok, changes have been made

**Comment by timvdstap** [comment]
This looks great, thank you so much for your work on this Matt! The only things I can think of - which might not be too pressing - would be:

- [ ] Include description text for 'Oceanographic', 'Biological' and 'Other' when you hover over it. 
- [ ] Is there any chance that the resource Type can show up in the recommended citation in the form as well? 
- [ ] Not sure if somehow 'Geographic Extent' can be made required (with an asterisk) when 'Biological' is selected?

**Pull request merged**